src/plugins/index_pattern_field_editor/public/components/preview/field_list/field_list_item.tsx

```diff
@@ -75,13 +75,19 @@ export const PreviewListItem: React.FC<Props> = ({
           <div
             className="indexPatternFieldEditor__previewFieldList__item__key__wrapper"
             data-test-subj="key"
+            aria-label={i18n.translate('indexPatternFieldEditor.fieldPreview.fieldName', {
+              defaultMessage: 'Field name',
+            })}
```

Comment:
`aria-label` on a `<div>` doesn't (consistently) work so we can remove this. But, also, if it did work, `aria-label` **replaces** the content of the element so the `key` (and `value` below) would never be read out which is also not great.

It looks like you're building a table with two columns. Do you think you could use a `<table>` instead of a bunch of `<div>` elements? The styles you're emulating are basically the same as **EuiTable**.

Reply:
I've removed the aria-label. 👍 

For the table I'd prefer to defer that as a future enhancement after FF. We want to make sure the style align, that there are no table borders nor table headers. I wonder if maybe a description list might be better? WDYT?